src/services/usersCRUD/UsersCRUD.go

```diff
@@ -0,0 +1,141 @@
+package usersCRUD
+
+import (
+	"net/http"
+	"github.com/gorilla/mux"
+	"strconv"
+	"regexp"
+	"log"
+	"github.com/Nastya-Kruglikova/cool_tasks/src/services/common"
+)
+
+type Users struct {
+	Id int
+	Login string
+	Password string
+	Name string
+}
+
+var users=[]Users {{1,"Karim","1234qwer","Karim"}}
+var user=Users{1,"Karim","1234qwer","Karim"}
+
+func SendUsers() ([]Users, error){
+	return users, nil
+}
+
+func ByID (id int)(user Users, err error) {
+	user=Users{1,"Karim","1234qwer","Karim"}
+	return user, err
+}
+
+func Add (user Users) error{
+	return nil
+}
+
+func Delete(id int) error {
+	return nil
+}
+
+func GetUsers (w http.ResponseWriter, r *http.Request) {
+	users, err:=SendUsers()
+	if err!=nil {
+		log.Print(err, " ERROR: Can't get users")
+		common.SendError(w, r, 404, "ERROR: Can't get users", err)
+		return
+	}
+	common.RenderJSON(w, r, users)
+}
+
+func GetUserByID (w http.ResponseWriter, r*http.Request) {
```

Comment:
Pls use gofmt (or setup it in your IDE)
Should be r *http.Request
space missed